.travis.yml

```diff
@@ -1,17 +1,13 @@
-sudo: false
```

Comment:
why are these removed here?

Reply:
Just to copy the rustc-guide Travis file closely.

Reply:
We don't need sudo though? This makes it stay on the container builds, which I think it defaults too so can we keep that in? :)